image/dicom_le.ksy

```diff
@@ -0,0 +1,147 @@
+  endian: le
+seq:
+  - id: file_header
+    type: t_file_header
+  - id: dataset
+    type: t_dataset
+types:
+  t_file_header:
+    seq:
+      - id: preamble
+        size: 128
+      - id: magic
+        contents: 'DICM'
+  t_dataset:
+    seq:
+      - id: elements1
+        type: t_dataentry_state_implicit
+        if: not _io.eof
+        repeat: until
+        repeat-until: _io.eof or _.is_transfer_syntax_change_explicit
+      - id: elements2
+        type: t_dataentry_state_explicit
+        repeat: eos
+  t_dataentry_state_implicit:
+    seq:
+      - id: tag_group
+        type: u2
+      - id: tag_elem
+        type: u2
+      - id: header
+        type: t_entry_header_implicit
+        if: entry_implicit
+      - id: header
+        type: t_entry_header_explicit
+        if: not entry_implicit
+      - id: content
+        size: header.content_length
+        if: has_content
+      - id: elements
+        type: t_dataentry_state_implicit
+        if: has_elements and not is_definite
+        repeat: until
+        repeat-until: _.is_closing_tag
+      - id: elements
+        type: t_elements
+        if: has_elements and is_definite
+        size: header.content_length
+    types:
+      t_elements:
+        seq:
+          - id: elements
+            type: t_dataentry_state_implicit
+            repeat: eos
+    instances:
+      entry_implicit:
+        value: tag_group != 2 or tag_group == 0xfffe
+      is_closing_tag:
+        value: tag_group == 0xfffe and tag_elem & 0xff00 == 0xe000 and tag_elem != 0xe000
+      has_elements:
+        value: (tag_group == 0xfffe and tag_elem == 0xe000) or header.is_seq
+      has_content:
+        value: not has_elements
+      is_definite:
+        value: header.content_length != 0xffffffff
+      p_is_transfer_syntax_change_explicit_1:
+        # '1.2.840.10008.1.2.1\0'
+        value: content == [49, 46, 50, 46, 56, 52, 48, 46, 49, 48, 48, 48, 56, 46, 49, 46, 50, 46, 49]
+      p_is_transfer_syntax_change_explicit_2:
+        value: content == [49, 46, 50, 46, 56, 52, 48, 46, 49, 48, 48, 48, 56, 46, 49, 46, 50, 46, 49, 0]
+      is_transfer_syntax_change_explicit:
+        value: tag_group == 2 and tag_elem == 16 and (p_is_transfer_syntax_change_explicit_1 or p_is_transfer_syntax_change_explicit_2)
+
+  t_dataentry_state_explicit:
+    seq:
+      - id: tag_group
+        type: u2
+      - id: tag_elem
+        type: u2
+      - id: header
+        if: header_is_implicit
+        type: t_entry_header_implicit
+      - id: header
+        if: not header_is_implicit
+        type: t_entry_header_explicit
+      - id: content
+        size: header.content_length
+        if: has_content
+      - id: elements
+        type: t_dataentry_state_explicit
+        if: has_elements and not is_definite
+        repeat: until
+        repeat-until: _.is_closing_tag
+      - id: elements
+        type: t_elements
+        if: has_elements and is_definite
+        size: header.content_length
```

Comment:
Everything related to duplicate `header` and `elements` applies to this type as well.